src/esp/scene/Mp3dSemanticScene.cpp

```diff
@@ -73,12 +75,13 @@ std::string Mp3dObjectCategory::name(const std::string& mapping) const {
   }
 }
 
-int Mp3dRegionCategory::index(const std::string& mapping) const {
+int Mp3dRegionCategory::index(CORRADE_UNUSED const std::string& mapping) const {
```

Comment:
Not sure if this matters or not, but we usually put this macro in the header.

Reply:
@mosra Does it matter?

Reply:
Should be just 

```cpp
int Mp3dRegionCategory::index(const std::string&) const {
```

in all cpp files, and `CORRADE_UNUSED` in headers (so the parameter name is kept for documentation purposes)